metomi/rose/suite_engine_procs/cylc.py

```diff
@@ -16,21 +16,23 @@
 # -----------------------------------------------------------------------------
 """Logic specific to the Cylc suite engine."""
 
-from glob import glob
 import os
 import pwd
 import re
-from random import shuffle
 import socket
 import sqlite3
 import tarfile
+
+from glob import glob
+from random import shuffle
+from typing import Union, List, Tuple, Any
 from time import sleep
 from uuid import uuid4
 
-from metomi.rose.fs_util import FileSystemEvent
-from metomi.rose.popen import RosePopenError
-from metomi.rose.reporter import Reporter
-from metomi.rose.suite_engine_proc import (
+from metomi.rose.fs_util import FileSystemEvent  # type:ignore
+from metomi.rose.popen import RosePopenError  # type:ignore
+from metomi.rose.reporter import Reporter  # type:ignore
+from metomi.rose.suite_engine_proc import (   # type:ignore
```

Comment:
Rose isn't setup for mypy yet.

Reply:
Are you telling me that I shouldn't do this? Or something else?

Reply:
As it stands MyPy isn't supposed to run for this code so mypy will always fail (explicit package bases) and every single Metomi import will cause errors.